sdk/aqueduct/table_artifact.py

```diff
@@ -156,6 +172,93 @@ def save(self, config: SaveConfig) -> None:
             ],
         )
 
+    def validate_with_expectation(
+        self,
+        expectation_name: str,
+        expectation_args: Optional[Dict[str, Any]] = None,
+        severity: CheckSeverity = CheckSeverity.WARNING,
+    ) -> CheckArtifact:
+        """Creates a check that validates with the table with great_expectations and its set of internal expectations.
+        The expectations supported can be found here:
+        https://great-expectations.readthedocs.io/en/latest/reference/glossary_of_expectations.html
+        The expectations supported are only those that can support Pandas on the great_expectations backend.
+        E.g. Use a expectation to check all column values are unique.
+        ge_check = table.validate_with_expectation("expect_column_values_to_be_unique", {"column": "fixed_acidity"})
+        ge_check.get() // True or False based on expectation passing
+        Args:
+            expectation_name:
+                Name of built-in expectation to run with great_expectations
+            expectation_args:
+                Dictionary of args to pass into the expectation suite for the expectation being run.
+            severity:
+                Optional severity associated with the check created with this expectations
+        Returns:
+            A check artifact that represent the validation result of running the expectation provided on the table.
+        """
+
+        def great_expectations_check_method(table: pd.DataFrame) -> bool:
+            data_context_config = DataContextConfig(
+                datasources={
+                    "my_pandas_datasource": DatasourceConfig(
+                        class_name="PandasDatasource",
+                    )
+                },
+                store_backend_defaults=FilesystemStoreBackendDefaults(root_directory="/tmp"),
+            )
+
+            context = BaseDataContext(project_config=data_context_config)
+
+            # create and load Expectation Suite
+            expectation_suite_name = "aq_check_expectation"
+            context.create_expectation_suite(
+                expectation_suite_name=expectation_suite_name, overwrite_existing=True
+            )
```

Comment:
Doesn't `create_expectation_suite` return the suite already? Why do you need `get_expectation_suite()`?

Reply:
you're right, i was snipping code from the ge docs and missed this